x-pack/legacy/plugins/siem/server/lib/detection_engine/notifications/rules_notification_alert_type.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Logger } from 'src/core/server';
+import { schema } from '@kbn/config-schema';
+import { i18n } from '@kbn/i18n';
+import { NOTIFICATIONS_ID } from '../../../../common/constants';
+
+import { NotificationAlertTypeDefinition } from './types';
+import { getSignalsCount } from './get_signals_count';
+import { RuleAlertAttributes } from '../signals/types';
+import { scheduleNotificationActions } from './schedule_notification_actions';
+
+export const rulesNotificationAlertType = ({
+  logger,
+  kibanaUrl,
+}: {
+  logger: Logger;
+  kibanaUrl: string;
+}): NotificationAlertTypeDefinition => ({
+  id: NOTIFICATIONS_ID,
+  name: 'SIEM Notifications',
+  actionGroups: [
+    {
+      id: 'default',
+      name: i18n.translate(
+        'xpack.siem.detectionEngine.ruleNotificationAlert.actionGroups.default',
+        {
+          defaultMessage: 'Default',
+        }
+      ),
```

Comment:
I thought there wasn't any `i18n` on the server side -- are we able to do this now, and if so, do you know how the locale gets correctly set?